examples/siro_sandbox/controllers/fetch_baselines_controller.py

```diff
@@ -508,6 +521,13 @@ def act(self, obs, env):
                         self.current_state = FetchState.PICK
                     self._init_policy_input()
 
+            # Check if the human blocks the robot when robot is near the target
+            self.human_block_robot_when_searching = (
```

Comment:
I can't find where  you're using `self._human_block_robot_when_searching`. Is this dead code?

Reply:
maybe searching for `self.human_block_robot_when_searching` ?